recipes/quickfix/all/test_package/conanfile.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class QuickfixTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"ssl": [True, False], "fPIC": [True, False]}
+    default_options = "ssl=False", "fPIC=True"
```

Comment:
I think options here are not relevant for the test package as they are not being used later, but as said later, we can improve it later